controller/include/ACV.h

```diff
@@ -0,0 +1,55 @@
+/* Copyright 2020, Edwin Chiu
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#ifndef ACV_H
+#define ACV_H
+
+/****************************************************************************************
+ *  @brief
+ *  @usage
+ *  @param
+ *  @param
+ *  @return
+ ****************************************************************************************/
```

Comment:
Ok, I'll remove them. Are we not planning on using doxygen ?

Reply:
Is there a document with this in it so that I may refer to it? I admit I prefer my code to be somewhat less dense as I find it easier to read.

Reply:
> Are we not planning on using doxygen ?

No objection to doxygen if someone wants to set it up, but until we have a comment ready for a function, paying 6 lines for an empty comment is a lot in my and @jkff's opinions.  See also  https://en.wikipedia.org/wiki/YAGNI.

> Is there a document with the coding standards in it so that I may refer to it? 

We can use https://google.github.io/styleguide/cppguide.html#Vertical_Whitespace if you want, that's been carefully put together.